flows/count_family_document_concepts.py

```diff
@@ -80,15 +80,22 @@ async def partial_update_family_document_concepts_counts(
     concepts_counts_with_names: dict[str, int],
     vespa_search_adapter: VespaSearchAdapter,
 ) -> None:
-    """Update document concept counts in Vespa via partial updates."""
+    """
+    Update document concept counts in Vespa via partial updates.
```

Comment:
comment: Nice that this effectively already has the update/delete behaviour (e.g. removing old count values)!